charts/argo-cd/values.yaml

```diff
@@ -1934,29 +1900,47 @@ server:
     # -- Automount API credentials for the Service Account
     automountServiceAccountToken: true
 
+  # Argo CD server ingress configuration
   ingress:
     # -- Enable an ingress resource for the Argo CD server
     enabled: false
-    # -- Additional ingress annotations
-    annotations: {}
+    # -- Specific implementation for ingress controller. One of `generic`, `aws` or `gke`
+    ## Additional configuration might be required in related configuration sections
+    controller: generic
```

Comment:
@mkilchhofer Any opinion on this parameter name that enables specific implementation for backing ingress controller (GKE, AWS maybe something else in the future that have kinda unique setup)

Reply:
Also gave some thoughts about this but didn't found a better name 😁 
names that came in to my mind:
- controller
- type
- flavour
- variant

But I'd say, lets go with `controller`. 👍